package.json

```diff
@@ -8,14 +8,15 @@
   "author": "GWI",
   "license": "BSD-3-Clause",
   "scripts": {
-    "test": "npx elm-verify-examples --elm-test=./node_modules/.bin/elm-test && npx elm-test",
+    "test": "npx elm-test",
+    "verify-examples": "npx elm-verify-examples --elm-test=./node_modules/.bin/elm-test",
     "test:w": "node ./node_modules/chokidar-cli/index.js \"**/*.elm\" -c \"npm run test\""
   },
   "devDependencies": {
     "chokidar-cli": "1.2.1",
     "elm": "0.19.0-no-deps",
     "elm-format": "^0.8.1",
-    "elm-test": "0.19.0-beta11",
-    "elm-verify-examples": "3.0.1"
+    "elm-test": "0.19.0-rev6",
+    "elm-verify-examples": "3.1.0"
```

Comment:
missing `^`? The other repo has it

Reply:
It's because I edited this one manually and created that one via yarn init. Nevermind, added those ^ here as well.